src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -281,7 +281,8 @@
           short-summary: Agent pool vm set type. VirtualMachineScaleSets or AvailabilitySet.
         - name: --enable-pod-security-policy
           type: bool
-          short-summary: (PREVIEW) Enable pod security policy.
+          short-summary: Enable pod security policy.
+          long-summary: --enable-pod-security-policy is deprecated. See https://aka.ms/aks/psp for details.
```

Comment:
Because the help message is hidden, maybe we can remove the help message directly